www/src/CPGraphQlTypes/ChargifySubscriptionInputType.php

```diff
@@ -12,22 +12,34 @@ class ChargifySubscriptionInputType
     private string $payment_token;
     private ChargifyAddressInput $shipping_address;
     private ChargifyAddressInput $billing_address;
+    private ?string $vat_number;
 
-    public function __construct(string $plan_handle, string $payment_token, ChargifyAddressInput $address)
-    {
+    public function __construct(
+        string $plan_handle,
+        string $payment_token,
+        ChargifyAddressInput $address,
+        ?string $vat_number = null
+    ) {
         $this->plan_handle = $plan_handle;
         $this->payment_token = $payment_token;
         $this->shipping_address = $address;
         $this->billing_address = $address;
+        $this->vat_number ??= $vat_number;
```

Comment:
`??=` wow, look at that, #til :D

Reply:
Ya know, I love `??=` but you mentioning it here reminds me that I don't think there's any situation where I would need it here. It's basically "is this set? if not, then go with this other thing" but this is in the constructor.